volumes/vds/readme.md

```diff
@@ -23,39 +24,106 @@
 
 ## Data Availability
-Tables `vds.raw_vdsdata` and `vds.raw_vdsvehicledata` and all subsequent summary tables (`counts_15min`, `counts_15min_bylane`, `veh_length_15min`, `veh_speeds_15min`) have data from 2021-11-01 and beyond pulled from ITSC using the new process described here.
-Data for table `vds.counts_15min` before 2021-11 was backfilled from the `rescu` schema, and only for certain columns. No other tables contain data before 2021-11.  
+- Tables `vds.raw_vdsdata` and `vds.raw_vdsvehicledata` and related summary tables (`counts_15min`, `counts_15min_bylane`, `veh_length_15min`, `veh_speeds_15min`) have data from 2021-11-01 and beyond pulled from ITSC using the new process described here.  
+- Data for table `vds.counts_15min` before 2021-11 was backfilled from the `rescu` schema, and only for certain columns. No other tables contain data before 2021-11.
+
 
 ## Future Work 
-See Issue #658 which will add additional data quality checks to the new schema. 
+- Future work is planned under issue #658 to add data quality checks to the new schema. 
+- Further investigation of new sensor classes (division_id 8001, BlueCity, SmartMicro) to determine utility  
+
 
 # Table Structure  
 ## vds.raw_vdsdata
-This table contains parsed data from ITSC public.vdsdata. 
-Column `volume_veh_per_hr` stores the volumes in vehicles per hour for that bin. Note that different sensors have different bins which affects the conversion from volume to count. To convert to 15 minute counts, see `vds.counts_15min` or the corresponding insert script at `bdit_data-sources/volumes/vds/sql/insert/insert_counts_15min.sql`. This method assumes both missing bins and zero values are zeros, in line with old pipeline. 
+This table contains parsed data from ITSC public.vdsdata.  
+Column `volume_veh_per_hr` stores the volumes in vehicles per hour for that bin. Note that different sensors have different bin width which affects the conversion from volume to count (see `vds.detector_expected_bins`). For details on converting `raw_vdsdata` to 15 minute counts, see `vds.counts_15min` or the corresponding insert script at `bdit_data-sources/volumes/vds/sql/insert/insert_counts_15min.sql`. This method assumes both missing bins and zero values are zeros, in line with old pipeline.  
 This table retains zero bins to enable potential future differente treatment of missing and zero values. 
-Contains `division_id IN (2, 8001)`. A one day sample for `division_id = 8001` was explored under the heading [vds.raw_vdsdata_div8001](#vdsraw_vdsdata_div8001); more investigation is needed to determine what this data can be used for. 
+Contains `division_id IN (2, 8001)`. A one day sample for `division_id = 8001` was explored under the heading [vds.raw_vdsdata_div8001](#vdsraw_vdsdata_div8001); more investigation is needed to determine what this data can be used for.  
```

Comment:
not sure if this link `[vds.raw_vdsdata_div8001](#vdsraw_vdsdata_div8001)` is working as intended